scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```diff
@@ -17,10 +17,10 @@ spec:
   selector:
     trustedservices.intel.com/cache: pccs
   ports:
-  - name: pccs
-    protocol: TCP
-    port: 8042
-    targetPort: pccs-port
+    - name: pccs
+      protocol: TCP
+      port: 8042
+      targetPort: pccs-port
```

Comment:
I see a bunch of YAML format fixes that seems unrelated. Please don't mix such cosmetic changes with real functional changes. Please move all the reformatting to a pre-liminary commit in this PR.